tests/git-scripts/pre-push.sh

```diff
@@ -32,18 +32,9 @@ echo "URL is $URL"
 
```

Comment:
Preexisting: this script runs on what's currently checked out, not on what's being pushed.

Reply:
Indeed. I'd rather consider it out of scope for this PR and address it independently.